canonical/ubuntu-pro-client
# i18n prep: refactor: move help text into code (SC-1597)

## Why is this needed?

Strings are more easily internationalized as python strings than as conffiles (also help_data.yaml shouldn't have been a conffile)

## Test Steps


Make sure upgrade removes the conffile appropriately.

Make sure `pro help <service>` subcommand still works.

## Checklist

 - [n/a] Changes here need to be documented, and this was done in: 

## Does this PR require extra reviews?
